src/components/MAPI/organizations/AccessControl/index.tsx

```diff
@@ -34,14 +33,13 @@ const AccessControl: React.FC<IAccessControlProps> = ({
 }) => {
   const orgNamespace = getOrgNamespaceFromOrgName(organizationName);
 
-  const client = useHttpClient();
-  const user = useSelector(getLoggedInUser);
+  const clientFactory = useHttpClientFactory();
```

Comment:
Now using a client factory instead of a single client, since multiple requests that run in parallel have different request configurations. Somehow this wasn't a problem before, but it now is 😬